docs/source/interfaces/Frequency Domain Analysis.rst

```diff
@@ -53,8 +53,12 @@ appropriate grouped workspace.
 Maximum Entropy Method
 ^^^^^^^^^^^^^^^^^^^^^^
 
-The maximum entropy method can be used to calculate the frequency domain spectrum. The MaxEnt Property table contains 
-the basic quantitites used during the calculation. The 
+The maximum entropy method can be used to calculate the frequency domain spectrum. This interface uses the :ref:`MuonMaxent <algm-MuonMaxent>` algorithm to calculate the frequency spectrum. The MaxEnt Property table contains 
+the basic quantitites used during the calculation. If the ``Use Phase Table`` checkbox is set to true, then an input phase table will be used for the initial phase values (the phase table will be called ``PhaseTable``). To generate
+an estimate phase table the ``Construct Phase Table`` option can be used and when the ``Calculate MaxEnt`` button is pressed ref:`CalMuonDetectorPhases <algm-CalMuonDetectorPhases>` is ran first to generate the phase table. If the 
```

Comment:
Missing `:` in `ref:`.

Reply:
I will add the : before ref with the fix for the failing test